.rosinstall

```diff
@@ -25,4 +25,7 @@
     local-name: src/mapping_urc/multi_resolution_graph
     uri: 'https://github.com/garethellis0/multi_resolution_graph.git'
     version: 0.0.3
+- git:
+    local-name: src/external_libs/evdev
+    uri: https://gitlab.freedesktop.org/libevdev/libevdev
```

Comment:
Did you end up successfully finding a way to install evdev via `rosdep`/`apt`? If so, it would probably be more desirable, otherwise this is fine.

Reply:
I will ask @kevinlinxc to try something out. If that works out, I can remove above lines

Reply:
Based on [this](https://github.com/UBC-Snowbots/Snowflake/pull/381#issuecomment-725225185), you'll be able to remove these lines?

Reply:
yep, not needed anymore